packages/@aws-cdk/aws-glue-alpha/README.md

```diff
@@ -223,7 +223,25 @@ new glue.Table(this, 'MyTable', {
 });
 ```
 
-By default, an S3 bucket will be created to store the table's data and stored in the bucket root. You can also manually pass the `bucket` and `s3Prefix`:
+Glue tables can be configured to contain user-defined properties, to describe the physical storage of table data, through the `storageParameters` property:
+
+```ts
+declare const myDatabase: glue.Database;
+new glue.Table(this, 'MyTable', {
+  storageParameters: [
+    { key: glue.StorageParameters.SKIP_HEADER_LINE_COUNT, value: glue.StorageParameterValue.custom('1') },
+    { key: glue.StorageParameters.COMPRESSION_TYPE, value: glue.StorageParameterValue.compressionType(glue.CompressionType.GZIP) },
+    { key: glue.StorageParameters.custom('separatorChar'), value: glue.StorageParameterValue.custom(',') }, // Will describe the separator char used in the data
+  ],
```

Comment:
I really don't like this API. I see the benefits of an array and why we can't use a map. However I'd like to simplify this alot:

```suggestion
  storageParameters: [
    glue.StorageParameter.skipHeaderLineCount(1), // note this is a number. The static method should convert it to a string.
    glue.StorageParameter.compressionType(glue.CompressionType.GZIP),
    glue.StorageParameter.custom('separatorChar', ',')
  ],
```

Which will reduce the type to something like
```ts
  storageParameters?: StorageParameter[];
```

And the implementation of `StorageParameter` to be a class with a private constructor that take a `key` and a `value`. It can only be instantiated by the defined set of public static methods.

Reply:
That's actually a lot better. Didn't think of that, I'll work on it.

Reply:
This is done